batch/test-locally.sh

```diff
@@ -13,6 +13,7 @@ server_pid=$!
 
 until curl -fL 127.0.0.1:5000/jobs >/dev/null 2>&1
 do
+    ((tries++)) && [ $tries -lt 30 ]
```

Comment:
Does this actually cause the loop to fail if tries is greater than 30. I think it would just return False.

Reply:
It is a shell command that returns a non-zero exit code which fails the shell because I previously executed `set -ex`. The docs from the bash man page explain:

```
 -e      Exit immediately if a simple command (see SHELL GRAMMAR above) exits
         with a non-zero status.  The shell does not exit if the command that
         fails is part of the command list immediately following a while or
         until keyword, part of the test in an if statement, part of a && or ||
         list, or if the command's return value is being inverted via !.  A trap
         on ERR, if set, is executed before the shell exits.
```

Reply:
(I also tested it to make sure ;) )